Python/executor.c

```diff
@@ -31,6 +31,39 @@
 #define ENABLE_SPECIALIZATION 0
 
 
+// TODO: continue editing this from tupleobject.c to fit _PyUOpExecutorObject
+static _PyUOpExecutorObject *
+tuple_alloc(Py_ssize_t size)
+{
+    _PyUOpExecutorObject *op = maybe_freelist_pop(size);
```

Comment:
freelists are an optimization. You could remove this line for now (and the branch that checks for `NULL` below

Reply:
Thank you for the help!!